### PR TITLE
Added possibility of overriding the body highlight style and the header title style of a node

### PR DESCRIPTION
**Pull request by fernando-toigo-hoplon**

### CODE REVIEW

File: Scripts/Editor/NodeEditor.cs [hunk 1]
```diff
@@ -100,6 +100,14 @@ public class NodeEditor : XNodeEditor.Internal.NodeEditorBase<NodeEditor, NodeEd
             return NodeEditorResources.styles.nodeBody;
         }
 
+        public virtual GUIStyle GetBodyHighlightStyle() {
+            return NodeEditorResources.styles.nodeHighlight;
+        }
+
+        public virtual GUIStyle GetHeaderTitleStyle() {
```
Comment (Siccity) [inline Scripts/Editor/NodeEditor.cs]: You have full access to override any aspect of the header drawing already. Redundant.

Comment (fernando-toigo-hoplon) [reply Scripts/Editor/NodeEditor.cs]: You can only override the whole drawing of the header, not just its style.
So, should I override and copy the base drawing of the header just to change a property of its style?

Comment (Siccity) [reply Scripts/Editor/NodeEditor.cs]: OnHeaderGUI looks like this
```cs
public virtual void OnHeaderGUI() {
    GUILayout.Label(target.name, NodeEditorResources.styles.nodeHeader, GUILayout.Height(30));
}
```
To change just the style you can change it before drawing it.
```cs
public override void OnHeaderGUI() {
    GUIStyle backup = NodeEditorResources.styles.nodeHeader; // Backup style
    NodeEditorResources.styles.nodeHeader = myCoolStyle; // Apply your style
    base OnHeaderGUI(); // Draw default header with your new style
    NodeEditorResources.styles.nodeHeader = backup; // Return style
}
```
Alternatively, you can redraw the entire thing with just one line
```cs
public override void OnHeaderGUI() {
    GUILayout.Label(target.name, myCoolStyle, GUILayout.Height(30));
}
```

Comment (fernando-toigo-hoplon) [reply Scripts/Editor/NodeEditor.cs]: Oh ok, I see what you mean. Thanks for the instructions.
So, I'll add a new commit to the PR removing the header part, is that ok?

Comment (Siccity) [reply Scripts/Editor/NodeEditor.cs]: Yes, thank you

**Pull request merged**